website/src/pages/covid/compare-side-by-side.astro

```diff
@@ -1,130 +1,6 @@
 ---
-import GsAggregate from '../../components/genspectrum/GsAggregate.astro';
-import GsMutations from '../../components/genspectrum/GsMutations.astro';
-import GsPrevalenceOverTime from '../../components/genspectrum/GsPrevalenceOverTime.astro';
-import GsRelativeGrowthAdvantage from '../../components/genspectrum/GsRelativeGrowthAdvantage.astro';
-import { CompareSideBySidePageStateSelector } from '../../components/pageStateSelectors/CompareSideBySidePageStateSelector';
-import { CompareSideBySideSelectorFallback } from '../../components/pageStateSelectors/FallbackElement';
-import { toDownloadLink } from '../../components/views/compareSideBySide/toDownloadLink';
-import { isStaging, getDashboardsConfig, getLapisUrl } from '../../config';
-import OrganismViewPageLayout from '../../layouts/OrganismPage/OrganismViewPageLayout.astro';
-import { chooseGranularityBasedOnDateRange } from '../../util/chooseGranularityBasedOnDateRange';
-import { ComponentHeight } from '../../views/OrganismConstants';
-import { toLapisFilterWithoutVariant } from '../../views/pageStateHandlers/toLapisFilterWithoutVariant';
-import type { OrganismViewKey } from '../../views/routing';
-import { ServerSide } from '../../views/serverSideRouting';
-
-const organismViewKey: OrganismViewKey = 'covid.compareSideBySideView';
-const view = ServerSide.routing.getOrganismView(organismViewKey);
-
-const pageState = view.pageStateHandler.parsePageStateFromUrl(Astro.url);
-
-const downloadLinks = [...pageState.filters.entries()].map(
-    toDownloadLink(view.pageStateHandler, view.organismConstants.organism),
-);
+import GenericCompareSideBySidePage from '../../components/views/compareSideBySide/GenericCompareSideBySidePage.astro';
+import { Organisms } from '../../types/Organism';
 ---
 
-<OrganismViewPageLayout view={view} downloadLinks={downloadLinks}>
-    <gs-app lapis={getLapisUrl(view.organismConstants.organism)}>
-        <div class='flex overflow-x-auto'>
-            {
-                Array.from(pageState.filters).map(([id, { variantFilter, datasetFilter }]) => {
-                    const baselineLapisFilter = toLapisFilterWithoutVariant(
-                        datasetFilter,
-                        view.organismConstants.additionalFilters,
-                    );
-                    const timeGranularity = chooseGranularityBasedOnDateRange({
-                        earliestDate: new Date(view.organismConstants.earliestDate),
-                        dateRange: datasetFilter.dateFilters[view.organismConstants.mainDateField],
-                    });
-                    const numeratorFilter = {
-                        ...variantFilter.lineages,
-                        ...variantFilter.mutations,
-                        variantQuery: variantFilter.variantQuery,
-                        ...baselineLapisFilter,
-                    };
```

Comment:
This was another difference to the generic page - but I think the logic is already handled by this:
https://github.com/GenSpectrum/dashboards/blob/ae2e48eb4fc6d19e7094a95fbecff474b9118069/website/src/views/pageStateHandlers/PageStateHandler.ts#L14-L28